prelude/src/Data/Reflectable.js

```diff
@@ -0,0 +1,5 @@
+// module Data.Reflectable
+
+export const unsafeCoerce = function (arg) {
```

Comment:
Do we need this definition for scheme?

Reply:
Ah yes, I typed it down and forgot to commit it